### DISCUSSION TITLE
Modal + Form + rangepicker，给rangepicker赋初始值时会陷入死循环报错

### DISCUSSION THREAD

**Discussion by lsjxscs** [ant-design/ant-design#45213] [Q&A]
在使用Modal + Form时，给rangepicker赋初始值时会陷入死循环报错
![image](https://github.com/ant-design/ant-design/assets/54137857/1aed9f9a-c6df-4e7e-81dc-8495956d6371)
![image](https://github.com/ant-design/ant-design/assets/54137857/b3f07e6c-aac6-4e80-b2b0-4bfbe3acdc98)
Maximum update depth exceeded
如果不使用modal的情况，则可以正常赋值
I don't konw why,help!!

**Comment by yoyo837** [comment]
提供复现

**Reply by lsjxscs** [reply]
![image](https://github.com/ant-design/ant-design/assets/54137857/13b80f5c-dbbe-470c-8865-ad48a3114be8)
![image](https://github.com/ant-design/ant-design/assets/54137857/30ac3d7a-41a6-4132-a46d-390cdbb58082)
就是基本的initialValues，用form.setfieldsvalues也不行

**Comment by Yuiai01** [comment]
用的是 moment 还是 dayjs 呢？

**Reply by lsjxscs** [reply]
moment

**Reply by Yuiai01** [reply]
V5 版本默认使用 dayjs，如果需要使用 moment 可以参考 https://ant.design/docs/react/use-custom-date-library-cn#datepicker.

**Reply by lsjxscs** [reply]
用了dayjs，正常了，感谢！
